pkgs/top-level/ruby-packages.nix

```diff
@@ -2202,6 +2202,14 @@
       type = "gem";
     };
     version = "0.7.1";
+    dontBuild = false;
+    postPatch = ''
+      # https://gitlab.com/libvirt/libvirt-ruby/-/commit/43543991832c9623c00395092bcfb9e178243ba4
```

Comment:
Please fetchpatch the commit instead.

Reply:
There are commits prior to this commit that don't let it be applied cleanly. Should I cherry-pick it?

Beside this, I need to move these modifications to `pkgs/development/ruby-modules/gem-config/default.nix`